### ISSUE TITLE
Discussions - keyboard accessibility

### ISSUE THREAD

**Issue by smendez92** [pubpub/pubpub#324]
It should be possible to select and expand/collapse discussions via Tab/Enter keys

**Comment by isTravis** [comment]
Thanks so much for reporting this and the other accessibility issues you've bumped into. We also have a report from an internal MIT accessibility specialist outlining a list of issues. We're planning to tackle these and make the needed architecture changes to properly address them in the next month or so. We'll update here when fixes have been pushed live.

**Comment by gabestein** [comment]
Discussions should now be keyboard accessible!
